src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -311,69 +314,100 @@ const Hole::ThreadDescription Hole::threadDescription[][171] =
+        { "#6",         3.505,  0.635,    2.95 },
+        { "#8",         4.166,  0.706,    3.50 },
+        { "#10",        4.826,  0.794,    4.10 },
+        { "#12",        5.486,  0.907,    4.70 },
+        { "1/4",        6.350,  0.907,    5.50 },
+        { "5/16",       7.938,  1.058,    6.90 },
+        { "3/8",        9.525,  1.058,    8.50 },
+        { "7/16",       11.113, 1.270,    9.90 },
+        { "1/2",        12.700, 1.270,    11.50 },
+        { "9/16",       14.288, 1.411,    12.90 },
+        { "5/8",        15.875, 1.411,    14.50 },
+        { "3/4",        19.050, 1.588,    17.50 },
+        { "7/8",        22.225, 1.814,    20.40 },
+        { "1",          25.400, 2.117,    23.25 },
+        { "1 1/8",      28.575, 2.117,    26.50 },
+        { "1 1/4",      31.750, 2.117,    29.50 },
+        { "1 3/8",      34.925, 2.117,    32.75 },
+        { "1 1/2",      38.100, 2.117,    36.00 },
     }    ,
     /* UNEF */
     {
-        { "#12",        5.4864, 0.7938 },
-        { "1/4",        6.3500, 0.7938 },
-        { "5/16",       7.9375, 0.7938 },
-        { "3/8",        9.5250, 0.7938 },
-        { "7/16",       11.1125, 0.9071 },
-        { "1/2",        12.7000, 0.9071 },
-        { "9/16",       14.2875, 1.0583 },
-        { "5/8",        15.8750, 1.0583 },
-        { "3/4",        19.0500, 1.2700 },
-        { "7/8",        22.2250, 1.2700 },
-        { "1",          25.4000, 1.2700 },
+        { "#12",        5.486,  0.794,    4.80 },
+        { "1/4",        6.350,  0.794,    5.70 },
+        { "5/16",       7.938,  0.794,    7.25 },
+        { "3/8",        9.525,  0.794,    8.85 },
+        { "7/16",       11.113, 0.907,   10.35 },
+        { "1/2",        12.700, 0.907,   11.80 },
+        { "9/16",       14.288, 1.058,   13.40 },
+        { "5/8",        15.875, 1.058,   15.00 },
+        { "11/16",      17.462, 1.058,   16.60 },
+        { "3/4",        19.050, 1.270,   18.00 },
+        { "13/16",      20.638, 1.270,   19.60 },
+        { "7/8",        22.225, 1.270,   21.15 },
+        { "15/16",      23.812, 1.270,   22.70 },
+        { "1",          25.400, 1.270,   24.30 },
+        { "1 1/16",     26.988, 1.411,   25.80 },
+        { "1 1/8",      28.575, 1.411,   27.35 },
+        { "1 1/4",      31.750, 1.411,   30.55 },
+        { "1 5/16",     33.338, 1.411,   32.10 },
+        { "1 3/8",      34.925, 1.411,   33.70 },
+        { "1 7/16",     36.512, 1.411,   35.30 },
+        { "1 1/2",      38.100, 1.411,   36.90 },
+        { "1 9/16",     39.688, 1.411,   38.55 },
+        { "1 5/8",      41.275, 1.411,   40.10 },
+        { "1 11/16",    42.862, 1.411,   41.60 },
     }
 
 };
 
-const double Hole::metricHoleDiameters[35][4] =
+const double Hole::metricHoleDiameters[36][4] =
 {
-    /* ISO metric according to ISO 273 */
+    /* ISO metric clearance hole diameters according to ISO 273 */
     // {screw diameter, close, standard, coarse}
         { 1.0,      1.1,    1.2,    1.3},

```

Comment:
Is it correct that this list lacks entries for M1.1, M2.2, M9, and M11? (I'm just comparing it to the list in `threadDescription`)

Reply:
Yes, the ISO 273 I could find does not define these values I could also not find them by googling around for excerpts of the norm. We only store values that are normed, so no suggestions you might find.